scripts/verify-replace.sh

```diff
@@ -16,27 +19,45 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo}).XXX)
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
     if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+        ERRORREPOLIST+="($(basename ${repo}))"
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies &> ${ERRFILE} 
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERRORREPOLIST+="($(basename ${repo}))" 
```

Comment:
Should there be a trailing space here, too, as it is in the `ERRORFILELIST`?

Reply:
i added the trailing space now , i didn't do it previously, as my thought was to have all the repo names printed in a single line, but i guess its better to have them in 1 line each